layout/column.ftd

```diff
@@ -23,6 +23,8 @@ optional integer bottom:
 optional integer left:
 optional integer right:
 optional integer z-index:
+string list css:
+string list js:
```

Comment:
Nopes. Why?

Reply:
@amitu This was added by Rithik in the main branch. I have added this because it was creating conflict.